lib/reversetunnel/remotesite.go

```diff
@@ -87,6 +87,8 @@ type remoteSite struct {
 
 	// nodeWatcher provides access the node set for the remote site
 	nodeWatcher *services.GenericWatcher[types.Server, readonly.Server]
+	// gitServerWatcher provides the Git server set for the remote site
+	gitServerWatcher *services.GenericWatcher[types.Server, readonly.Server]
```

Comment:
The resource watcher will also constantly spew errors and reset if the underlying event stream can't be initialized because the leaf auth server doesn't know about the existence of git_server.